Makefile

```diff
@@ -13,6 +13,11 @@ test-integration:
 	terraform --version
 	go test -v -run=CLI ./test
 
+test-go-licenses:
+	cd .. && go version && go get github.com/google/go-licenses
```

Comment:
Does using a `cd ..` here mean that the following line will run in the parent directory or will the working directory reset?

Reply:
Each line within make is sandboxed. So it won't affect the subsequent line.

https://stackoverflow.com/questions/1789594/how-do-i-write-the-cd-command-in-a-makefile